_posts/2023-06-15-our-journey-at-f5-with-apache-arrow-part-2.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: "Our journey at F5 with Apache Arrow (part 2)"
```

Comment:
I wonder if including some specifics about the content of this post in the title would help people find it more readily. Something like the following perhaps. This is just a suggestion and at least one potential reason to leave the current title is for consistency with part 1. 

```suggestion
title: "Our journey at F5 with Apache Arrow (part 2): Adaptive Schemas and Sorting to Optimize Memory Usage"
```

Reply:
Good idea. However I will replace to "Optimize Memory Usage" with "Optimize Arrow Usage" as the sorting approach is mainly optimizing the compression ratio.

Reply:
Done